scripts/hak/target.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.isHost = exports.isHostId = exports.getHost = exports.ENABLED_TARGETS = exports.TARGETS = void 0;
+const aarch64AppleDarwin = {
```

Comment:
Sorry, perhaps I should have been more explicit, but this file was generated by `tsc` from the TS version in `element-desktop`. It might make a bit more sense after looking at https://github.com/vector-im/element-builder/pull/42, but perhaps I need some comments as well...

Reply:
I have added a warning in the file to make it more obvious that it has been generated from something elsewhere.